src/components/IncidentInfo/IncidentInfo.js

```diff
@@ -22,11 +33,15 @@ const IncidentInfo = ({ id }) => {
         {console.log(incident)}
         <p data-cy='info-name' className='info info-name'>{incident.name}</p>
         <p data-cy='info-type' className='info type'><span className='info-type'>{getIconByType(incident.incident_type)}</span>{incident.incident_type}</p>
-        <a href={mapURL} target="_blank" rel="noopener noreferrer" className='map-btn'>Click to get Map to Headquarters</a>
+        <div className="info-button-container">
+          <a href={mapURL} target="_blank" rel="noopener noreferrer" className='map-btn'>Click to get Map to Headquarters</a>
+          {!incident.close_date && <button className="end-button" onClick={e => handleSubmit(e)}>Declare Incident Over</button>}
```

Comment:
Might be more readable to use incident.active here, like so:
`{incident.active && <button className="end-button" onClick={e => handleSubmit(e)}>Declare Incident Over</button>}`